sql/hive/src/main/scala/org/apache/spark/sql/hive/execution/SaveAsHiveFile.scala

```diff
@@ -210,12 +210,21 @@ private[hive] trait SaveAsHiveFile extends DataWritingCommand {
       stagingDir)
   }
 
+  /**
+   * This method should only be called in test code.
+   */
+  def getStagingDirTest(
```

Comment:
invokePrivate?

Reply:
Many members are undefined in `SaveAsHiveFile` trait, so  object creation impossible for it .

Reply:
Then why not just `private[hive] def getStagingDir`?

Reply:
OK, it looks better, I will change it, thanks